.github/workflows/sanitizer.yml

```diff
@@ -0,0 +1,58 @@
+name: C/C++ Sanitizer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  job1:
+    name: adress-sanitizer
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: lukka/get-cmake@latest    
+    - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
the `ilammy/msvc-dev-cmd` does not harm, but it is not necessary if you do not build with VS (i.e. msvc) afterwards. It could be removed in this line only.

Reply:
Thanks for the hint, fixed!